src/main/java/com/team4/testingsystem/services/impl/QuestionServiceImpl.java

```diff
@@ -72,4 +78,11 @@ public Question updateQuestion(Question question, Long id) {
         return questionRepository.getQuestionsByTestId(id);
     }
 
+    @Transactional
+    @Override
+    public void archiveQuestionsByContentFileId(Long id) {
+        ContentFile contentFile = contentFilesRepository.findById(id)
```

Comment:
Let's archive content file

Reply:
Theoretically, yes.
In practice, the listening test is generated by a chosen content file. If a file is archived, the questions will never be found.
Alternatively, you can make a single request, that will archive all the questions by its content file. But definitely we don't need to make 10 requests for it